### PR TITLE
docs(jsdoc): fix jsdoc generation (missing files, constructor hack)

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

**Comment by bobylito** [comment]
LGTM :)

File: docs/_includes/widget-jsdoc/InstantSearch.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-| <span class='attr-optional'>`options.searchParameters`</span><span class="attr-infos">Type: <code>string</code></span> | Initial search configuration. |
-| <span class='attr-optional'>`options.urlSync`</span><span class="attr-infos">Type: <code>string</code></span> | Url synchronization configuration. |
-| <span class='attr-optional'>`options.urlSync.trackedParameters`</span><span class="attr-infos">Type: <code>string</code></span> | Parameters that will be synchronized in the URL. By default, it will track the query, all the refinable attribute (facets and numeric filters), the index and the page. |
-| <span class='attr-optional'>`options.urlSync.useHash`</span><span class="attr-infos">Type: <code>string</code></span> | If set to true, the url will be hash based. Otherwise, it'll use the query parameters using the modern history API. |
-| <span class='attr-optional'>`options.urlSync.threshold`</span><span class="attr-infos">Type: <code>string</code></span> | Time in ms after which a new state is created in the browser history. The default value is 700. |
-
-<p class="attr-legend">* <span>Required</span></p>
```
Comment (redox) [inline docs/_includes/widget-jsdoc/InstantSearch.md]: This file has been removed?

Comment (vvo) [inline docs/_includes/widget-jsdoc/InstantSearch.md]: Yes it was duplicated when ran on a linux machine. It was not here in your original PR.

Comment (vvo) [inline docs/_includes/widget-jsdoc/InstantSearch.md]: I fixed all of it (should)

**Pull request merged**